modules/social_features/social_group/social_group.install

```diff
@@ -1001,3 +1001,19 @@ function social_group_update_11401(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Disable "Get the related groups for this entity".
+ *
+ * Disable "Get the related groups for this entity" field on "Private Message
+ * Author" display for "User" entity if it is enabled.
+ */
+function social_group_update_11402(): void {
```

Comment:
@zanvidmar If we cherry-pick this to e.g. `11.3.x` then `social_group_update_11401` won't execute where those sites will be updated to `11.4.x`. It would probably be better to move this update to the `post_update` file whether do that by hook alter.